src/utils/cors.ts

```diff
@@ -0,0 +1,23 @@
+import { BrowserWindow } from 'electron';
+
+function lowerCaseHeaders(responseHeaders: Record<string, string[]>) {
+    const headers: Record<string, string[]> = {};
+    for (const key in responseHeaders) {
+        headers[key.toLowerCase()] = responseHeaders[key];
+    }
+    return headers;
+}
+
+export function addAllowOriginHeader(
+    mainWindow: BrowserWindow
+) {
+    mainWindow.webContents.session.webRequest.onHeadersReceived(
```

Comment:
this looks very hacky and shady 😬 . will check once what's the issue. we should ideally not need this.